specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2021-06-01/cluster.json

```diff
@@ -1195,6 +1195,10 @@
         "fileshare": {
           "type": "string",
           "description": "The file share name."
+        },
+        "enableSecureChannel" : {
```

Comment:
You add one property in the "StorageAccount" which is referred by both create request body and response body this is  a breaking change.

Reply:
If we only hope one property is only request body, we can mark it is "x-ms-mutability": ["create", "update"], but I am not sure if this still is breaking change because the response body refer this structure too.

@leni-msft could you please help confirm this? Thank you very much!

Reply:
This new property is optional for request and will not show in response.

Reply:
@jianyexi to help confirm

Reply:
This is a breaking change which isn't captured by the tool, you need to bump the api version if you want to add optional field to the request.

Reply:
Thanks @jianyexi .